src/main/resources/application.properties

```diff
@@ -1,8 +1,8 @@
 server.port=8081
```

Comment:
revert this file changes and commit. 
Port, context-path and datasource url should be same as  dev branch.

Reply:
Resolved